README.md

```diff
@@ -28,22 +28,69 @@ There aren't very many tests as this is a very simple app.
 
     rails -t
 
-## Deployment
+### Dependent gems
 
-To mimic running the application in a deployed state you can run
-`make image` and this will run through the assets precompilation, linting and testing before creating a Docker image. To view the Docker container you can run `make run`
+As of 2022-04-04, most of the local (i.e. Epimorphics) gems that this project
+depends on are served via GitHub Package Registry (GPR). Specifically,
+`lr_common_styles` and `json-rails-logger`. However, `qonsole-rails` is **not**
+served via GPR at present, mostly because we are hoping to retire it in favour
+of a new implementation of Qonsole. Since `qonsole-rails` is a public repo,
+this dependency does not require us to lean on the old pattern of using an ssh
+key to serve private gems directly from a GitHub repo.
 
-To bypass the need for running locally AWS you can pass a global variable to the command with `ECR=local make image`
+Accessing gems from GPR will require a personal access token PAT. To store this
+locally, use `make auth`. To create a PAT, see [the Epimorphics
+wiki](https://github.com/epimorphics/internal/wiki/Ansible-CICD#creating-a-pat-for-gpr-access).
 
-You can run `make help` to view a list of other make commands available
+### Pre-flight testing
 
-## Entrypoint.sh
+To mimic running the application in a deployed state you can run `make image`
+and this will run through the assets precompilation, linting and testing before
+creating a Docker image. To view the Docker container you can run `make run`
 
-* The Rails Framework requires certain values to be set as a Global environment variable when starting. To ensure the `RAILS_RELATIVE_URL_ROOT` is only set in one place per application we have added this to the Entrypoint file along with the `SCRIPT_NAME`.
-* The Rails secret is also created here.
-* There is a workaround to removing the PID lock of the Rails process in the event of the application crashing and not releasing the process.
-* We have to pass the `API_SERVICE_URL` so that it is available in the Entrypoint.sh or the application will throw an error and exit before starting
+To bypass the need for running locally AWS you can pass a global variable to
```

Comment:
Should that read something like "To bypass the need for AWS, and run locally, you ..."

Reply:
Thanks. That wasn't my content (my autoformatter changed the line lengths, which is why they're in the diff). In fact, reading it now it's out-of-date: `make image` doesn't require AWS anyway. `make publish` does, but I can't see a logical way to publish locally :smile: 

I removed those lines.